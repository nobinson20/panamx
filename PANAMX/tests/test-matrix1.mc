int main() {
    int i;
    int j;
    matrix<int>[3][2] m;
    matrix<double>[2][3] a;

    m = [11, 22; 33, 44; 55, 66;];
    for (i = 0; i < 3; i++) {
        for (j = 0; j < 2; j++) {
            print(m[i][j]);
        }
    }

    a = [1.1, 2.2, 3.3; 4.4, 5.5, 6.6;];
    for (i = 0; i < 2; i++) {
        for (j = 0; j < 3; j++) {
            printf(a[i][j]);
        }
    }
    return 0;
}
