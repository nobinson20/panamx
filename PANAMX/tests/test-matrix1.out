11
22
33
44
55
66
1.1
2.2
3.3
4.4
5.5
6.6
